src/objects/Batch_Template__c.object

```diff
@@ -0,0 +1,294 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We should probably pull the template object out for now. Or maybe that's hard enough to refactor around that we should have a conversation about its future first?

Reply:
Given meeting today, keeping it in for now.